jetty-core/jetty-server/src/main/java/org/eclipse/jetty/server/handler/ErrorHandler.java

```diff
@@ -262,6 +261,13 @@ else if (charsets.contains(StandardCharsets.ISO_8859_1))
         }
     }
 
+    protected int getBufferSize(Request request)
```

Comment:
We need a bit of javadoc to say that this is getting the buffer size for the entire error response (any larger responses will be truncated).   Then say that default implementation imposes an 8K limit on the standard buffer size, but that implementations may override the method to provide some other size.